### PR TITLE
Update naming format of PowerShell scripts - functions part

### PR DESCRIPTION
**Pull request by v-sijliu**
Run all BVT cases against Azure - Pass
Pick up 2 cases run against Hyper-V - Pass

### CODE REVIEW

**Comment by juhlee-microsoft** [comment]
@LiliDeng @v-sijliu I don't recall we decided to rename it yet. The change is not an issue but you need to buy team's agreement. 
suggest to buy team's agreement before moving forward.

File: Libraries/CommonFunctions.psm1 [hunk 1]
```diff
@@ -89,60 +89,60 @@ function Write-Log()
...
 }
 
-Function LogErr($text)
+Function Write-LogErr($text)
 {
 	Write-Log "ERROR" $text
 }
 
-Function LogError($text)
+Function Write-LogError($text)
```
Comment (lizzha) [inline Libraries/CommonFunctions.psm1]: This one is duplicated, can be removed, and replace all the occurrences with Write-LogErr

**Comment by juhlee-microsoft** [comment]
@v-sijliu The changes were simple but happened many placed. Can you randomly select and run tests if changes are not breaking? As far as it does not break, I will sign off.

**Comment by LiliDeng** [comment]
@lizzha Rebase latest code, run 3 BVT cases against Azure - pass, run 1 BVT case against Hyper-V - pass

**Pull request merged**